gtk4/src/subclass/widget.rs

```diff
@@ -1,9 +1,9 @@
 // Take a look at the license at the top of the repository in the LICENSE file.
 
+use crate::prelude::*;
 use crate::{
     AccessibleRole, BuilderScope, DirectionType, LayoutManager, Orientation, Shortcut,
     SizeRequestMode, Snapshot, StateFlags, SystemSetting, TextDirection, Tooltip, Widget,
-    WidgetExt,
 };
 use glib::prelude::*;
 use glib::subclass::{prelude::*, SignalId};
```

Comment:
glib prelude should already be pulled in by `crate::prelude::*`? And maybe the subclass prelude can be used as well through `crate::subclass::prelude::*`.

Reply:
Good point! let me update those :)